src/cluster/websocket/WebSocket.cc

```diff
@@ -144,7 +144,7 @@ WebSocketClient::SendInfo WebSocketClient::SendError(std::string_view tag, std::
     *out++ = ',';
     broker::format::json::v1::append_field("code", tag, out);
     *out++ = ',';
-    broker::format::json::v1::append_field("message", message, out);
+    broker::format::json::v1::append_field("context", message, out);
```

Comment:
Thanks! I remember being a bit annoyed by `"context"` when picking up the code and apparently changed it to `"message"`. Agree strongly on keeping `"context"`. I'll mark this for 7.2, too.